windows_create_package.bat

```diff
@@ -37,30 +37,34 @@ if !errorlevel! neq 0 exit /b !errorlevel!
 copy C:\%output_user_root%\openvino\runtime\bin\intel64\Release\*.dll dist\windows\ovms
 if !errorlevel! neq 0 exit /b !errorlevel!
 
+set "dest_dir=C:\opt"
+
+:: Remove below if Windows should always come without python
```

Comment:
Lets add a simple parameter to the script so that we can set one variable inside script and python will be added.
Can we remove install python from build dependencies now or we want to keep it for python nodes ?